client/memez/projects/memez/src/app/components/post/post.component.html

```diff
@@ -36,18 +36,22 @@
   <div class="post-content">
     {{post.content}}
   </div>
-  <div class="like-dashboard">
-    <div class="like-btn">
-      <mat-icon (click)="onPostLiked(post)"
-                class="like-icon">favorite
-      </mat-icon>
+  <ng-container *mobxAutorun>
+    <div class="like-dashboard"
+    >
+      <div class="like-btn">
+        <mat-icon (click)="onPostLiked(post)"
+                  class="like-icon">favorite
+        </mat-icon>
 
+      </div>
+      <div class="likes-amount"
```

Comment:
you can move the *mobxAutorun here because it is the one holding observales in it